content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/carousel.html

```diff
@@ -46,14 +46,16 @@
             <button class="cmp-carousel__action cmp-carousel__action--previous"
                     type="button"
                     aria-label="${'Previous' @ i18n}"
-                    data-cmp-hook-carousel="previous">
+                    data-cmp-hook-carousel="previous"
```

Comment:
I think the original issue was not as much about play/pause buttons being after prev/next but rather that they are after the carousel items. IMO, the tab indexes should be setup so that on component focus you can access the controls in following order:
1. play/pause
2. autoplay
3. prev/next
4. carousel items

Reply:
Using positive tabindex is an anti-pattern. I think we need changes in markup to solve this issue, so it would require a new version of the Carousel component.

Reply:
As a workaround for the current version, does it make sense to disable just the carousel items and keep all the controls focusable?

Reply:
If we disable the disable the tab navigation for the carousel items it would not be possible to pause the animation via tab navigation on item focus.

Reply:
We could change the markup if we add a content policy/design option that allows you to move them before the items (if not set, by default they will still be behind, as they are now).

Reply:
That would be backwards-compatible. I think we originally decided to place them after the items for SEO reasons. Then users can decide what's more important: accessibility or SEO.

Reply:
It sounds pretty hacky to implement in this version. I would prefer if we implement this correctly in the new version

Reply:
As discussed with @vladbailescu and @gabrielwalt I added a content policy option to control the placements of the controls in 630795c37451e5c52a240c3bbca6ada9ce095df0. The default value is backward compatible.